neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -215,5 +217,27 @@ public virtual bool Verify(Snapshot snapshot, IEnumerable<Transaction> mempool)
             if (net_fee < 0) return false;
             return this.VerifyWitnesses(snapshot, net_fee);
         }
+
+        public StackItem ToStackItem()
+        {
+            return new VM.Types.Array
+            (
+                new StackItem[]
+                {
+                    new Integer(Version),
+                    new Integer(Nonce),
+                    new ByteArray(Sender.ToArray()),
+                    new Integer(SystemFee),
+                    new Integer(NetworkFee),
+                    new Integer(ValidUntilBlock),
+                    // Attributes
+                    // Cosigners
+                    new ByteArray(Script),
+                    new VM.Types.Array(Witnesses.Select(u=>new ByteArray(u.VerificationScript))),
```

Comment:
Perhaps we should remove the `Witnesses` from `ToStackItem()` according to #1029

Reply:
Fully agree Erik, this is the moment to fix this. No more Witness access :muscle: